dashboard/app/controllers/registrations_controller.rb

```diff
@@ -386,8 +387,8 @@ def get_users_to_destroy(user)
     user.dependent_students << user.summarize
   end
 
-  def destroy_dependent_users(user)
-    user_ids_to_destroy = get_users_to_destroy(user).pluck(:id)
```

Comment:
After this change, `get_users_to_destroy` is a one-liner called in only one place.  Maybe we should get rid of it?

Reply:
good call! i'll update with that change